### ISSUE TITLE
Accessibility: Talkback does not read the value of some elements

### ISSUE THREAD

**Issue by RastislavKish** [woheller69/whisperIME#91]
Hello, a blind user here. :)

## Reproduction steps

1. Open the app on its main screen.
2. Turn on Talkback by long pressing the up and down volume keys.
3. use left / right swipes to move around the screen.

## Expected behavior

After the labels denoting the selected model and language, the screenreader should read the spinners with their selected values. Also, on the TextView with the result text, it should read the content of the component.

## Actual behavior

The screenreader can find the spinners as well as the TextView, but only reads their description, not their value.

## Explanation

The reason for this bug is the usage of the contentDescription attribute in the activity layout file. contentDescription is typically used on static widgets, where the screenreader has no way of determining how to meaningfully report the widget to the user. For example, the record button and the other buttons on the screen are a perfect use-case for this attribute, because they're just icon buttons, and the screenreader is unable to communicate their purpose represented by the embedded images.

However, for components carying a dynamic value, like the spinners and the result TextView, this value is what screenreaders need to report to the user, so the contentDescription is not necessary. In fact, it is actually counterproductive, since it overrides the component dynamic value, which the user needs to access. In these scenarios, VI users can determine the purpose of the component by its adjacent label, which is accessible for the screenreaders.

## Solution

Removing the contentDescription attribute from the spinners and the result TextView makes thing work really nicely on my phone. :)

I could open a PR, but it's really just 3 line change.

Thanks for the great job!

**Comment by woheller69** [comment]
I guess this is what you mean 

https://github.com/woheller69/whisperIME/commit/651d663526f09bf4210f93bfad6fb6d88962fe0f

**Comment by RastislavKish** [comment]
Exactly, thank you!
